nameko/containers.py

```diff
@@ -383,7 +403,7 @@ def _run_worker(self, worker_ctx, handle_result):
                 del exc_info
 
                 self.dependencies.all.worker_teardown(worker_ctx)
-                self.dependencies.injections.all.release(worker_ctx)
+                # release?
```

Comment:
did we have an intended use-case? i.e. some extension that wanted to use a dependency in another extensions teardown? think we ended up just using teardown tidy up from `acquire_injection`